ApplicationPattern.yaml

```diff
@@ -681,6 +695,16 @@ paths:
                           description: >
                             'Own release number
                             from [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-number]'
+                        preceding-application-name:
+                          type: string
```

Comment:
Same comment as above https://github.com/openBackhaul/ApplicationPattern/pull/484/files#r1056512131

Reply:
Many thanks for indicating.
Beware of changed semantics in the new proposal.